src/wagtailmedia/views/chooser.py

```diff
@@ -112,7 +112,9 @@ def chooser(request, media_type=None):
                 "pagination_template": pagination_template,
                 "media_type": media_type,
                 "ordering": ordering,
-                "linkurl": reverse("wagtailmedia:chooser"),
+                "linkurl": reverse("wagtailmedia:chooser_typed", args=({media_type}))
```

Comment:
this should have `args=(media_type,)`

and the definition is repeated, so makes sense to extract.
Will fix locally and merge